src/components/pages/observation/list/views/stats/table-uploaders.tsx

```diff
@@ -36,30 +40,22 @@ export default function UploadersTable({ data, title, loadMoreUniqueSpecies, fil
             {data.list.map(({ name, pic, authorId, count }) => (
               <Tr key={`${authorId}-${count}`}>
                 <Td>
-                  <HStack spacing="10px">
+                  <HStack spacing={5}>
                     <Avatar position="relative" src={getUserImage(pic, 50)} name={name} />
-                    <ExternalBlueLink
-                      href={
-                        filter?.groupName
-                          ? `/group/${filter.groupName}/user/show/${authorId}`
-                          : `/user/show/${authorId}`
-                      }
-                    >
-                      {name}
-                    </ExternalBlueLink>
+                    <LocalLink href={`/user/show/${authorId}/`} prefixGroup={true}>
+                      <ExternalBlueLink>{name}</ExternalBlueLink>
+                    </LocalLink>
                   </HStack>
                 </Td>
 
                 <Td>
-                  <ExternalBlueLink
-                    href={
-                      filter?.groupName
-                        ? `/group/${filter?.groupName}/observation/list?${address}&user=${authorId}`
-                        : `/observation/list?${address}&user=${authorId}`
-                    }
+                  <LocalLink
+                    href={`/observation/list`}
+                    params={{ ...queryParams, view: "list", user: authorId }}
```

Comment:
if you are going to override `user` anyway then there's no need to destructure it above just override it here